Source/WebCore/Modules/audiosession/DOMAudioSession.cpp

```diff
@@ -105,7 +110,12 @@ DOMAudioSession::Type DOMAudioSession::type() const
     if (document && !PermissionsPolicy::isFeatureEnabled(PermissionsPolicy::Feature::Microphone, *document, PermissionsPolicy::ShouldReportViolation::No))
         return DOMAudioSession::Type::Auto;
 
-    return document ? document->topDocument().audioSessionType() : DOMAudioSession::Type::Auto;
+    if (document) {
```

Comment:
If there's no document, this could just early return immediately instead of checking it twice and indenting code.